openapi2kong/openapi2kong.go

```diff
@@ -713,6 +713,12 @@ func Convert(content []byte, opts O2kOptions) (map[string]interface{}, error) {
 	//
 	//
 
+	if doc.Paths == nil {
+		doc.Paths = &v3.Paths{
+			PathItems: orderedmap.New[string, *v3.PathItem](),
+		}
+	}
```

Comment:
or is it better to fail with an actionable error message?

Reply:
I think it would be better to fail fast. There's not much we can do if there are no paths provided

Reply:
updated, lmk if it looks good now